### PR TITLE
feat(simple-rest): install required packages after swizzle

### PR DESCRIPTION
**Pull request by alicanerdurmaz**
feat: `requiredPackages` added to refine.config.js
Now with that, CLI automatically installs the required packages for the project.

chore: deprecated warning added to `stringify` export. Users should use from `query-string` package instead.

### CODE REVIEW

File: .changeset/brown-birds-stare.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@refinedev/simple-rest": minor
+---
+
+feat: `requiredPackages` added to refine.config.js
+Now with that, CLI automatically installs the required packages for the project.
```
Comment (BatuhanW) [inline .changeset/brown-birds-stare.md:6]:
```suggestion
feat: `requiredPackages` added to refine.config.js for CLI to automatically install required packages for the project.
```

File: packages/simple-rest/test/utils/generateFilter.spec.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        expect(result).toEqual({});
+    });
+
+    it("creates a filter object based on the provided filters", () => {
+        const filters: CrudFilters = [
+            { field: "name", operator: "eq", value: "John" },
+            { field: "age", operator: "gte", value: 25 },
+        ];
+        const result = generateFilter(filters);
+        console.log(result);
```
Comment (BatuhanW) [inline packages/simple-rest/test/utils/generateFilter.spec.ts:16]: ✏️

File: packages/simple-rest/test/utils/generateFilter.spec.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        },
+    );
+
+    it("creates a filter object with the 'q' field", () => {
+        const filters: CrudFilters = [
+            { field: "q", operator: "eq", value: "searchValue" },
+            { field: "name", operator: "eq", value: "John" },
+        ];
+        const result = generateFilter(filters);
+        console.log(result);
```
Comment (BatuhanW) [inline packages/simple-rest/test/utils/generateFilter.spec.ts:39]: ✏️

File: packages/simple-rest/test/utils/mapOperator.spec.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+        for (const operator in operatorMappings) {
+            const expectedResult = operatorMappings[operator as CrudOperators];
+            expect(mapOperator(operator as CrudOperators)).toEqual(
+                expectedResult,
+            );
+        }
+    });
+
+    it.each(["unsupported", "", undefined, null])(
```
Comment (BatuhanW) [inline packages/simple-rest/test/utils/mapOperator.spec.ts:43]: 🎖️

**Review by salihozdemir — APPROVED** [review]

**Pull request merged**